src/components/Player/Player.tsx

```diff
@@ -161,7 +161,13 @@ const Player: React.FC<Props> = ({
 
       // Player options are untyped
       const playerOptions: { [key: string]: unknown } = {
-        advertising: adScheduleData,
+        advertising: {
```

Comment:
Would it work if we change these settings in the default player instead of hardcoding showCountdown and showAdMarkers? Then customers will still have the flexibility to control them from the dashboard if they want.

Reply:
Never mind, those settings aren't in the dashboard